### PR TITLE
Add lacuna labels converter

### PR DESCRIPTION
**Pull request by ivorbosloper**
Implements https://github.com/fiboa/data/issues/44

### CODE REVIEW

File: fiboa_cli/datasets/lacuna_labels.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+resulting from a project to generate a continent-wide set of crop field
+labels for Africa covering the years 2017-2023. The data are intended
+for training and assessing machine learning models that can be used to
+map agricultural fields over large areas and multiple years.
+
+The project was funded by the [Lacuna Fund](https://lacunafund.org/),
+and led by [Farmerline](https://farmerline.co/), in collaboration with
+[Spatial Collective](https://spatialcollective.com/) and the
+[Agricultural Impacts Research Group](agroimpacts.info) at [Clark
+University](https://www.clarku.edu/departments/geography/).
```
Comment (m-mohr) [inline fiboa_cli/datasets/lacuna_labels.py:20]:
```suggestion
[Agricultural Impacts Research Group](agroimpacts.info) at
[Clark University](https://www.clarku.edu/departments/geography/).
```

File: fiboa_cli/datasets/lacuna_labels.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+Please refer to the [technical
+report](docs/report/technical-report.pdf) for more details on the
+methods used to develop the dataset, an analysis of label quality, and
+usage guidelines.
```
Comment (m-mohr) [inline fiboa_cli/datasets/lacuna_labels.py:25]:
```suggestion
Please refer to the [technical report](docs/report/technical-report.pdf)
for more details on the methods used to develop the dataset, an analysis
of label quality, and usage guidelines.
```

File: fiboa_cli/datasets/lacuna_labels.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+Data is published at https://zenodo.org/records/11060871 and can be used in accordance
+with Planet’s [participant license agreement for the NICFI
+contract](https://assets.planet.com/docs/Planet_ParticipantLicenseAgreement_NICFI.pdf).
```
Comment (m-mohr) [inline fiboa_cli/datasets/lacuna_labels.py:29]:
```suggestion
Data is published at https://zenodo.org/records/11060871 and can be used in accordance with
[Planet’s participant license agreement for the NICFI contract](https://go.planet.com/nicfi-pla-2024).
```

File: fiboa_cli/datasets/lacuna_labels.py [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+"""
+PROVIDERS = [
+    {
+        "name": "Agricultural Impacts Research Group",
+        "url": "https://github.com/agroimpacts/lacunalabels/",
+        "roles": ["producer"]
+    },
+    {
+        "name": "Planet Labs Inc.",
```
Comment (m-mohr) [inline fiboa_cli/datasets/lacuna_labels.py:38]: Inc is outdated, in recent contracts I always had to use: Planet Labs PBC

```suggestion
        "name": "Planet Labs PBC",
```

File: fiboa_cli/datasets/lacuna_labels.py [hunk 5]
```diff
@@ -0,0 +1,81 @@
+from ..convert_utils import convert as convert_
+
+
+SOURCES = "https://zenodo.org/records/11060871/files/mapped_fields_final.parquet?download=1"
+ID = "lacuna_labels"
```
Comment (m-mohr) [inline fiboa_cli/datasets/lacuna_labels.py:5]:
```suggestion
ID = "lacuna"
```

Maybe just lacuna?

File: fiboa_cli/datasets/lacuna_labels.py [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+        "name": "Planet Labs Inc.",
+        "url": "https://www.planet.com/",
+        "roles": ["licensor"]
+    }
+]
+ATTRIBUTION = "Planet Labs Inc."
+
+LICENSE = {
+    "title": "PLANET NICFI PARTICIPANT LICENSE AGREEMENT",
+    "href": "https://university.planet.com/nicfi-resources/1219786",
```
Comment (m-mohr) [inline fiboa_cli/datasets/lacuna_labels.py:47]:
```suggestion
    "title": "Planet NICFI participant license agreement",
    "href": "https://go.planet.com/nicfi-pla-2024",
```

**Review by m-mohr — APPROVED** [review]
A couple of smaller editorial suggestions, but nothing major.

File: fiboa_cli/datasets/lacuna_labels.py (hunk 5 above)
Comment (ivorbosloper) [reply fiboa_cli/datasets/lacuna_labels.py:5]: Do I need to update the source-coop repository name as well?

**Comment by m-mohr** [comment]
@ivorbosloper Was about to merge, but a changelog entry would be nice 😅

**Pull request merged**